docs/cloud-config.md

```diff
@@ -0,0 +1,68 @@
+# Description of cloud config
+
+## File Structure
+```
+[Global]
+cloud=          # myhuaweicloud.com
+auth-url=       # Auth url 
+region=         # Region where you control resourses
+access-key=     # AK Infomation
+secret-key=     # SK Infomation
+project-id=     # ProjectID in your region
+[Vpc]
+id=             # VPC where your cluster resides
+```
+
+## Introduction
+
+* Fields listed in the file are used to construct API request parameters, and complete identity authentication
+
+* A request URI is in the following format: **{URI-scheme}://{Endpoint}/{resource-path}?{query-string}** 
+
+### Global
+
+* `cloud`
+  Optional. Used to construct Endpoint. If not set, "myhuaweicloud.com" is used.

```

Comment:
We usually describe it as follows:
The endpoint of the cloud provider. Defaults to "myhuaweicloud.com".

Reply:
done